sdk/src/main/java/co/omise/android/models/BaseModel.kt

```diff
@@ -0,0 +1,8 @@
+package co.omise.android.models
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo
+import com.fasterxml.jackson.databind.annotation.JsonTypeIdResolver
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.CUSTOM, include = JsonTypeInfo.As.EXISTING_PROPERTY, property = "object", visible = true)
+@JsonTypeIdResolver(ModelTypeResolver::class)
```

Comment:
Is it possible to put this annotations to `Model`?

Reply:
From memory, I don't think so. But I'll double check and let you know

Reply:
@nuxzero it actually still works, so made the change 👌